pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -82,7 +84,7 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 	command.Flags().MarkHidden("max-scale")
 	p.markFlagMakesRevision("max-scale")
 
-	command.Flags().IntVar(&p.Scale, "scale", 0, "Minimum and maximum number of replicas.")
+	command.Flags().StringVar(&p.Scale, "scale", "1", "Minimum and maximum number of replicas.")
```

Comment:
I suggest to add a sample at the end of the explanation, e.g.
```
Minimum and maximum number of replicas. (e.g., 1..5).
```
in order to help end users understand the format clearly.